rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderBridgeTxsIndexTests.java

```diff
@@ -0,0 +1,354 @@
+import co.rsk.config.BridgeConstants;
+import co.rsk.config.BridgeMainNetConstants;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
+import org.ethereum.core.Repository;
+import org.ethereum.vm.PrecompiledContracts;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.util.stream.Stream;
+
+import static co.rsk.peg.BridgeStorageIndexKey.BRIDGE_BTC_TX_SIG_HASH;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class BridgeStorageProviderBridgeTxsIndexTests {
+
+    private static final String DUPLICATED_INSERTION_ERROR_MESSAGE = "Given bridge btc tx sigHash %s already exists in the index. Index entries are considered unique.";
+    private static final byte TRUE_VALUE = (byte) 1;
+    private final BridgeConstants bridgeConstants = BridgeMainNetConstants.getInstance();
+
+    private static Stream<Arguments> null_sigHash_parameters() {
+        return Stream.of(
+            Arguments.of(false),
+            Arguments.of(true)
+        );
+    }
+
+    private static Stream<Arguments> non_null_sigHash_parameters() {
+        return Stream.of(
+            Arguments.of(false, Sha256Hash.ZERO_HASH),
+            Arguments.of(false, PegTestUtils.createHash(10)),
+            Arguments.of(true, Sha256Hash.ZERO_HASH),
+            Arguments.of(true, PegTestUtils.createHash(20))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("null_sigHash_parameters")
+    void hasBtcTxSigHash_null_sigHash(boolean isRskip379HardForkActive) {
+        // Arrange
+        ActivationConfig.ForBlock activations = isRskip379HardForkActive?
+                                                    ActivationConfigsForTest.tbd600().forBlock(0) :
+                                                    ActivationConfigsForTest.fingerroot500().forBlock(0);
+
+        Repository repository = mock(Repository.class);
+
+        BridgeStorageProvider provider = new BridgeStorageProvider(
+            repository,
+            PrecompiledContracts.BRIDGE_ADDR,
+            bridgeConstants,
+            activations
+        );
+
+        // Act
+        boolean result = provider.hasBridgeBtcTxSigHash(null);
+
+        // Assert
+        Assertions.assertFalse(result);
+
+        verify(repository, never()).getStorageBytes(
+            any(),
+            any()
+        );
+
+        verify(repository, never()).addStorageBytes(
+            any(),
+            any(),
+            any()
+        );
```

Comment:
The any() Method Be More Explicit By Passing The Data Type Its Expecting.. any(RskAddress.class)

Reply:
Any() with explicit type exclude nulls. In this case, I'm testing using null values, and I  want to assert that no matters the value, none of those methods are called. 

For this case `any()` is the one that suits what it's being tested.